app/helpers/application_helper.rb

```diff
@@ -1,2 +1,23 @@
 module ApplicationHelper
+
+  def vote_count
+    self.votes.map {|vote| vote.value}.reduce(:+).to_s
```

Comment:
This should be in a module in models not in a view helper.
Also, you can do `votes.sum(:value)` using AR rather than loading all the votes and summing them in ruby.